web/content/js/nav.js

```diff
@@ -12,7 +12,7 @@ function buildNavigation() {
 
   return `<nav>
   <a href="/">
-    <img class="logo" src="/img/helmet.svg" alt="Home">
+    <img class="logo" src="/img/helmet.png" alt="Home">
```

Comment:
A note for future me/future someone: We want svg here.

Reply:
Wouldn't it be easier to add the text as real text in here instead of adding it to the image? That way search engines can also read it. Or is it about using a specific font?

Reply:
To answer your questions:

* I intended the helmet + text as a single clickable unit. This is supposed to be a link to the home page and I want to avoid a situation where only the image part is clickable and not the text.
* I took this https://github.com/theforeman/foreman-graphics/blob/master/logo/foreman_large.png as a template and worked my way from there.
* Also, yes, it's about using a specific font.

I hope that explains my intentions but truth be told, this is one of the things that I'm hoping to get @kmalyjur's input on once she and I are able to meet again.

I appreciate your feedback, please keep it coming, this is a good place to record any concerns that we should address.

Reply:
> I intended the helmet + text as a single clickable unit. This is supposed to be a link to the home page and I want to avoid a situation where only the image part is clickable and not the text.

You can do that with HTML too:
```html
  <a href="/">
    <img class="logo" src="/img/helmet.svg" alt="Home">
    Foreman Documentation
</a>
```

> Also, yes, it's about using a specific font.

You can load fonts into CSS and use a specific font too.

Reply:
Applied and used the font from https://github.com/theforeman/foreman-graphics/blob/master/stickers/ostrich-black.ttf